### DISCUSSION TITLE
Language Support for French

### DISCUSSION THREAD

**Discussion by ColinLubout** [SnosMe/awakened-poe-trade#1463] [Q&A]
Hi

I would like to add French support to the app. I have found the language files and just wanted to check if there is an easy way to generate them or do I just have to duplicate and manually update all the item data?

**Comment by Brimalion** [comment]
Hi,
I'm also interested in making French available for this awesome tool and I can offer my help for this.
I have found the language folder (currently 3 in master/renderer/public/data/ : en, ru ,kr) and I'm also wondering how to generate the [items.ndjson] and [stats.ndjson]
Thanks in advance

**Comment by meips** [comment]
Would it be possible to translate it into Spanish? Thank you for making this possible!
